ceph-build-os/config/definitions/ceph-build-os.yml

```diff
@@ -0,0 +1,79 @@
+- scm:
+    name: ceph-build
+    scm:
+      - git:
+          url: https://github.com/ceph/ceph-build.git
```

Comment:
doesn't look like you need to clone ceph-build, are you using anything in ceph-build?

Anything in the current job (ceph-build-os) can be "included", so no need to clone it